doc/docs/master/reference/scaling_limits.md

```diff
@@ -0,0 +1,56 @@
+# Pipeline Scaling Limits in Community Edition
+
+Pachyderm Community Edition **limits the number of pipelines** users can deploy and **caps the parallelism** settings of those pipelines.  
+
+If your DAGs require more than what our Community Edition allows you to run, [ask for a free trial and receive a time-limited enterprise license token](https://www.pachyderm.com/trial). 
+
+## Scaling Limits
+
+|**Number of pipelines** deployed| **Number of workers** for each pipeline|
+|------|------|
+|Community Users can deploy **up to 16 pipelines**.| Community Users can run **up to 8 workers in parallel** on each pipeline.|
+
+Are you hitting those limits? You are likely to be needing our assistance in optimizing your DAGs. We are here to help! See [Lift the limitation](#lift-the-limitation) and request a free trial of our Enterprise Edition.
```

Comment:
With the addition above we shouldn't need this anymore or the `Lift the Limitations` section, right?